app/controllers/publish/courses/base_funding_type_controller.rb

```diff
@@ -36,6 +38,8 @@ def funding_type
       def funding_type_fields
         raise NotImplementedError
       end
+
+      def goto_preview? = params.dig(funding_type, :goto_preview) == 'true'
```

Comment:
Feels like i've seen this a few times in the earlier PRs - do we need to DRY it up? i guess the only difference here is the form name?

Reply:
Yeah I was going to defer that for the last card so we can see all the duplication and variations